src/extensions.msal/Microsoft.Identity.Client.Extensions.Msal/Microsoft.Identity.Client.Extensions.Msal.csproj

```diff
@@ -3,17 +3,16 @@
   <PropertyGroup>
     <TargetFrameworkNetDesktop>net45</TargetFrameworkNetDesktop>
     <TargetFrameworkNetStandard>netstandard2.0</TargetFrameworkNetStandard>
-    <TargetFrameworkNetCore2>netcoreapp2.1</TargetFrameworkNetCore2>
-    <TargetFrameworkNetCore3>netcoreapp3.0</TargetFrameworkNetCore3>
```

Comment:
Looks like the extension was supposed to be built for .net core 2 and .net core 3, but there is no .net core 3 logic right now, so reverting this. Also, the flags were wrong. We need a lot of scrutiny for releasing the extension from here ...